src/nti/fakestatsd/tests/test_fakestatsd.py

```diff
@@ -78,7 +79,6 @@ def test_timer(self):
         assert_that(metric, instance_of(Timer))
         assert_that(metric.name, is_('glork'))
         assert_that(metric.value, is_('320'))
-        assert_that(metric, has_property('kind', is_('ms')))
```

Comment:
Why did this assertion go away? If it's because we always treat these as class attributes now, then it would probably be good to rename it to, e.g., `KIND`. Otherwise, they should probably stay.

Reply:
Yea that was my thought, but, as you point this out I realize that part of a future refactor will be to make these instance attributes of a standard `Metric` class so they need to go back in.